scripts/setup.js

```diff
@@ -50,17 +36,16 @@ document.addEventListener("keypress", function(){
 
       finder.on('file', (file) => {
         if (path.extname(file) == ".mp3") {
-          if(!state.nowplaying.instance){
-            state.allFiles.push(file)
-            state.load(state.allFiles[0])
-          }
-          else{
-            insertSorted(state.allFiles, file)
+          state.allFiles.push(file)
+          if(!state.nowplaying.instance) // For first time nowplaying.instance not exists so we check for it
+          {
+            state.load(state.allFiles[0]) // load first song
           }
         }
       })
 
       finder.on('end', async () => {
+        state.allFiles.sort()
```

Comment:
`allFiles` is a list of file paths and it's being sorted by file names when it is supposed be based on the song name.